fuse.go

```diff
@@ -1264,13 +1275,24 @@ func (r *InitRequest) Respond(resp *InitResponse) {
 	out.MaxReadahead = resp.MaxReadahead
 	out.Flags = uint32(resp.Flags)
 	out.MaxWrite = resp.MaxWrite
+	out.TimeGran = resp.TimeGran
+
+	if r.Kernel.GE(Protocol{7, 28}) {
+		out.Flags |= uint32(InitMaxPages)
```

Comment:
Wouldn't it be better to set the flag while initializing the mount instead of checking the fuse version for every response?

Reply:
I don't follow, the init response should only be generated once on mount.

Reply:
Sorry for the bother, I wasn't aware that it would be called only once. Thanks for the clarification.